.github/workflows/test.yml

```diff
@@ -23,6 +23,11 @@ jobs:
       - name: Lint
         run: npm run lint
 
+      - name: Addons Linter
+        run: |
+          npm run build -- firefox
```

Comment:
My only concern is to run `npm run build` multiple times during this action, but as I propose to add the build step to the `addons-linter` command, it must be like this.